src/configuration.cpp

```diff
@@ -718,7 +736,9 @@ std::string chimera::CompiledConfiguration::Lookup(const YAML::Node &node) const
 }
 
 bool chimera::CompiledConfiguration::Render(
-    std::string view, std::string key,
+    const std::string &key, const std::string &header_view,
+    const std::string &source_view, const std::string &header_ext,
+    const std::string &source_ext,
```

Comment:
Why do we pass in the `header_ext` and the `source_ext` if they are always the same values?  Can't we just specify them in this function when we call the other `Render()`?

Reply:
It makes sense to me. Done in https://github.com/personalrobotics/chimera/pull/289/commits/9f8d9b5a78d3cfa31640af349fcec377e09238d7.